pkg/registries/registries.go

```diff
@@ -4,20 +4,32 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/containers/image/pkg/sysregistriesv2"
+	"github.com/containers/image/v5/pkg/sysregistriesv2"
 	apioperatorsv1alpha1 "github.com/openshift/api/operator/v1alpha1"
 )
 
-// scopeMatchesRegistry returns true if a scope value (as in sysregistriesv2.Registry.Prefix / sysregistriesv2.Endpoint.Location)
-// matches a host[:port] value in reg.
-func scopeMatchesRegistry(scope, reg string) bool {
+// regMatchesScope returns true if a scope value (as in sysregistriesv2.Registry.Prefix / sysregistriesv2.Endpoint.Location)
+// matches a host[:port] value in reg or if it is a sub-scope of reg.
+func regMatchesScope(scope, reg string) bool {
```

Comment:
So is `reg` now only a host:port, or a full scope? If the latter, as it seems, the naming of everything is confusing. `scopeIsNestedInsideScope(subScope, superScope)`?

Reply:
Fixed